### PR TITLE
Minor update on activesupport

### PR DESCRIPTION
**Pull request by fede-moya**
Address [CVE-2020-8165](https://groups.google.com/g/rubyonrails-security/c/bv6fW4S0Y1c)

### CODE REVIEW

File: codeclimate.gemspec [hunk 1]
```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = "~> 2.5"
 
-  s.add_dependency "activesupport", ">= 5.1.3", "< 5.2"
+  s.add_dependency "activesupport", ">= 5.2.4.3", ">= 5.2"
```
Comment (filipesperandio) [inline codeclimate.gemspec:21]: Can this be simplified?
```suggestion
  s.add_dependency "activesupport", ">= 5.2"
```
Do we have any comments about why it was previously explicitly keeping it < 5.2?

Comment (fede-moya) [reply codeclimate.gemspec]: yeap, done ! 👍🏼

File: codeclimate.gemspec [hunk 2]
```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = "~> 2.5"
 
-  s.add_dependency "activesupport", ">= 5.1.3", "< 5.2"
+  s.add_dependency "activesupport", ">= 5.2"
```
Comment (filipesperandio) [inline codeclimate.gemspec:21]: sorry to bring this again, I think this should use the `Pessimistic Version Operator` (`~>`) like the others below.
Maybe something like `~> 5.2.4`, allowing it to apply new patches but without jumping minor versions, that might be problematic with activesupport I believe. WDYT?

Comment (fede-moya) [reply codeclimate.gemspec]: Actually that sounds like a clever idea. Done.

> sorry to bring this again
No, thanks for always find a way to improve the work.

**Review by frnico33 — APPROVED** [review]
LGTM!

**Pull request merged**